### ISSUE TITLE
[bug]: 3.0.0b5 Mac mps - Textual inversion training fails at start with python error - Accelerator __init__() got an unexpected keyword argument 'logging_dir'

### ISSUE THREAD

**Issue by panicsteve** [invoke-ai/InvokeAI#3763]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

mps

### What version did you experience this issue on?

3.0.0b5

### What happened?

Not sure if textual inversion training is meant to be implemented for mps in 3.0, but after selecting it from the invoke.sh menu, configuring the training, and starting it, it fails with a python error:

```
Textual inversion training
[2023-07-13 13:27:01,136]::[InvokeAI]::ERROR --> An exception occurred during training. The exception was:
[2023-07-13 13:27:01,136]::[InvokeAI]::ERROR --> __init__() got an unexpected keyword argument 'logging_dir'
[2023-07-13 13:27:01,137]::[InvokeAI]::ERROR --> DETAILS:
[2023-07-13 13:27:01,137]::[InvokeAI]::ERROR --> Traceback (most recent call last):
  File "/Volumes/Storage/AI/InvokeAI-3b5/.venv/lib/python3.9/site-packages/invokeai/frontend/training/textual_inversion.py", line 426, in do_front_end
    do_textual_inversion_training(InvokeAIAppConfig.get_config(),**args)
  File "/Volumes/Storage/AI/InvokeAI-3b5/.venv/lib/python3.9/site-packages/invokeai/backend/training/textual_inversion_training.py", line 587, in do_textual_inversion_training
    accelerator = Accelerator(
TypeError: __init__() got an unexpected keyword argument 'logging_dir'
```

Google led me to this issue on the huggingface accelerate repo, which may be relevant: https://github.com/huggingface/accelerate/issues/1559

**Comment by pharmpk** [comment]
I've had the same problem with b3, b4, b5 and 2.x. I tried a fix on an earlier thread without success.

**Comment by panicsteve** [comment]
This appears to be resolved in 3.0.0rc1.

Textual inversion training now runs on mps!  But it does default to "fp16" mixed precision, which will lead to a crash in mps if you don't change it to "no".  That's related to #3671, so I'll close this issue, and add to that one.
